StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
+    private String sfname;
+    private String slname;
+
+    StudentData(int sno, String sfname, String slname) {
+        this.sno = sno;
+        this.sfname = sfname;
+        this.slname = slname;
+    }
+
+    public int getSno() {
+        return sno;
+    }
+
+    public String getSlname() {
+        return slname;
+    }
+
+    public String getSfname() {
+        return sfname;
+    }
+
+    public String toString() {
+        return sno + " " + sfname + " " + slname;
+    }
+}
+
+class StudentDetails {
+    private static final int ADD_STUDENT = 1;
+    private static final int SHOW_STUDENT = 2;
+    private static final int SEARCH_STUDENT = 3;
+    private static final int REMOVE_STUDENT = 4;
+    private static final int UPDATE_STUDENT = 5;
+
+    public static void main(String[] args) {
+
+        List<StudentData> c = new ArrayList<StudentData>();
```

Comment:
What is __c__? __c__ has no meaning at all. Use appropriate names. Since it seems that __c__ should contain your students, just use the name __students__.